Skeleton/src/Order.cpp

```diff
@@ -1,14 +1,55 @@
 #include "../include/Order.h"
 
-Order::Order(int id, int customerId, int distance) : id(id), customerId(customerId), distance(distance) {
+Order::Order(int id, int customerId, int distance) : id(id), customerId(customerId), distance(distance) 
+{
     driverId = NO_VOLUNTEER;
     collectorId = NO_VOLUNTEER;
 }
-Order::Order(int id, int customerId, int distance, int collectorId, int driverId) : id(id), customerId(customerId), distance(distance), driverId(driverId), collectorId(collectorId) {} 
-int Order::getId() const {return id;}
-int Order::getCustomerId() const {return customerId;}
-void Order::setStatus(OrderStatus status) {status = status;}
+Order::Order(int id, int customerId, int distance, int collectorId, int driverId) 
+    : id(id), customerId(customerId), distance(distance), driverId(driverId), collectorId(collectorId) {} 
 
+int Order::getId() const 
+{
+    return id;
+}
+
+int Order::getCustomerId() const 
+{
+    return customerId;
+}
+
+void Order::setStatus(OrderStatus status) 
+{
+    status = status;
```

Comment:
maybe safer to use: this->status = status
we can change it only if it will make a problem

Reply:
relevant to all set parameters

Reply:
resolved in my new branch